OctopusDSC/Tests/cOctopusSeqLogger.Tests.ps1

```diff
@@ -97,20 +101,20 @@ try
                 }
 
                 It 'Throws an exception if Octopus is not installed and ensure is set to present' {
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe" }
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe.nlog" }
+                    Mock Get-OctopusServerExePath { return "" }
+                    Mock Get-TentacleExePath { return "" }
                     Mock Test-NLogDll { return $true }
                     Mock Get-NLogConfig { return  [xml] (Get-Content (Join-Path $sampleConfigPath "octopus.server.exe.nlog-with-valid-configuration-with-api-key.xml")) }
-                    { Get-TargetResourceInternal @desiredConfiguration } | Should -throw "Unable to find Octopus (checked for existence of file '$octopusServerExePath')."
+                    { Get-TargetResourceInternal @desiredConfiguration } | Should -throw "Unable to find Octopus (checked for existence of file '')."
                 }
 
                 It 'Does not throw an exception if Octopus is not installed and ensure is set to absent' {
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe" }
-                    Mock Test-Path { return $false } -ParameterFilter { $Path -eq "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe.nlog" }
+                    Mock Get-OctopusServerExePath { return "" }
```

Comment:
I think I prefer `Test-Path` to be mocked here... Otherwise we're mocking something one step away from the test, ie, relying on Test-Path to return false when passed an empty string.